packages/docusaurus-plugin-sitemap/src/types.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export interface PluginOptions {
```

Comment:
any reason to use interface not type here?

Reply:
Mostly for consistency with other plugins - .e.g. : https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-plugin-content-pages/src/types.ts or https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-plugin-content-blog/src/types.ts

Reply:
would need to bring to attention @yangshun @endiliey

Reply:
this is okay. interface makes it easier to extend and even using advanced types like Partial